edge-util/src/Microsoft.Azure.Devices.Edge.Util/Option.cs

```diff
@@ -140,6 +140,14 @@ public void ForEach(Action<T> action)
             }
         }
 
+        public void ForEach(Action action)
+        {
+            if (this.HasValue)
+            {
+                action();
+            }
+        }
+
         public void ForEach(Action<T> action, Action none)
```

Comment:
is it possible that 2nd parameter be optional with default value "empty action"?

Reply:
"Action none = () => { }" doesn't work because it needs to be compile-time constant, but I don't think that even fits the specific case here - the case where I need to perform an action if Option.HasValue, but I don't need the value as the parameter.